infrastructure/lambda/template.yaml

```diff
@@ -632,6 +636,52 @@ Resources:
 #                                                                  #
 ####################################################################
 
+  DLDrivingPermitCheckLambdaConcurrencyThresholdReached:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmDescription: !Sub Driving Permit CRI ${Environment} - DrivingPermitCheck Lambda concurrency threshold reached
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlarmTopicDL
+      OKActions:
+        - !Ref AlarmTopicDL
+      InsufficientDataActions: []
+      MetricName: ConcurrentExecutions
+      Namespace: AWS/Lambda
+      Statistic: Sum
+      Dimensions:
+        - Name: FunctionName
+          Value: !Ref DrivingPermitCheckingFunction
```

Comment:
Were you able to check is it possible to reference common from here?

Reply:
Amended to alarm for the common-api stack used by the api